extension/background/background.js

```diff
@@ -48,42 +48,31 @@ async function fetchChapters(videoId) {
 
 async function fetchChaptersFromComments(videoResponse) {
     const comments = await youtubei.fetchComments(videoResponse)
+    const pinnedComment = comments.find((comment) => comment.isPinned)
+    if (!pinnedComment) {
+        return []
+    }
 
+    const tsContexts = getTimestampContexts(pinnedComment.text)
     const minNumChapters = 2
 
-    for (let i = 0; i < comments.length; i++) {
-        if (!comments[i].isPinned) {
-            continue
-        }
-        const tsContexts = getTimestampContexts(comments[i].text)
-        if (tsContexts.length >= 2) {
-            return tsContexts
-        }
-    }    
-
-    return []
+    return (tsContexts.length >= minNumChapters) ? tsContexts : []
```

Comment:
Brackets aren't necessary here.

Reply:
I know. I often use round brackets in one-line ternary expressions, so that I can easily see the condition.